mono/mini/mini-amd64.c

```diff
@@ -8176,11 +8176,15 @@ mono_arch_emit_inst_for_method (MonoCompile *cfg, MonoMethod *cmethod, MonoMetho
 	int opcode = 0;
 
 	if (cmethod->klass == mono_defaults.math_class) {
+#if 0
+		/* Mono can emit Sin and Cos directly as fsin and fcos, but modern libc implementations should be faster. */
```

Comment:
This is not the way to do it. The code should be removed including on the backends.

How does this impact arm?